back/src/bsds/typeDefs/bsd.objects.graphql

```diff
@@ -61,3 +61,19 @@ type Signature {
   date: DateTime
   author: String
 }
+
+type RevisionsInfos {
+  "La révision a-t-elle déjà été révisée ?"
```

Comment:
Pas certain de comprendre, est-ce que ce n'est pas plutôt : 

```suggestion
  "La bordereau a-t-il déjà été révisé ?"
```

Reply:
Yes of course, une révision révisée ca ne veut pas dire grand chose. Je vais modifier le commentaire